docs/GettingStarted.md

```diff
@@ -0,0 +1,244 @@
+# Pace-Neutrons: getting started
+version: 0.3.0
+## Introduction
+
+Pace-Neutrons, previously known as pace-python, is a python package which brings Horace 3.6.3 and SpinW to python removing the need for Matlab whilst keeping the commands you have become familiar with.
+
+## Requirements
+
+- Windows
+- Python 3.8, 3.9 or 3.10 (You can check this by running `python --version` in your terminal)
+- Matlab R2021b runtime
+
+## Installation
+
+The easiest way to install `pace-neutrons` and its installer is by downloading the `pace_neutrons_installer` from the [release](https://github.com/pace-neutrons/pace-python/releases) on github.
+When this is run, it will install the pace-neutrons pypi package as well as the Matlab runtime required to use the package.
```

Comment:
Link to PyPI package

Reply:
We haven't done the full release yet. Duc wanted to wait until Horace 4 is released I believe.

Reply:
Doesn't matter too much, these docs will outlast this release.

Reply:
I can add the main pypi page then.